### ISSUE TITLE
Engineering Office Hours 5/13

### ISSUE THREAD

**Issue by luthermonson** [rancher/rancher#37721]
Rancher Backend Engineers @luthermonson and @Oats87 are going to hold a public Rancher Engineering Office Hours on May 13th at 10AM PST, 5PM GMT. Any open source contributors looking to merge their contributions or just curious look lous who want to get more details on Rancher2 architecture are welcome to attend.

We would like to keep this structured so if you wish to attend, please comment below with details of what you want out of the office hours and link PRs or Issues you'd like to discuss. This way we can develop an agenda and see if we need to add any additional SUSE/Rancher engineers to the meeting invite.

[Teams Link](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MGNkOTU5ZDUtZTg1Yy00MzI5LTkxNjYtNTExODA0ZDA4MzVj%40thread.v2/0?context=%7b%22Tid%22%3a%22f7a17af6-1c5c-4a36-aa8b-f5be247aa4ba%22%2c%22Oid%22%3a%22e8a95f86-335f-4af4-9f84-7928ae02fce1%22%7d)

**Comment by Shaked** [comment]
Unfortunately I missed this one. Fridays are weekends in my timezone. 

- Any plans of having more of these on different days? 
- The problem with GH issues is that it's hard to follow for specific updates regarding "Office hours". Any idea how to do that?
- Back in https://github.com/rancher/rancher/issues/37451#issuecomment-1109921826 we spoke about a diagram for how Rancher TLS works. Is there a plan to share it? 

Thank you @luthermonson!

**Comment by Oats87** [comment]
The next office hours issue is here: https://github.com/rancher/rancher/issues/37889
